lab-davidw/app/filter/fuzzy-search.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = function() {
+  return function(galleries, searchTerm) {
+    if(!searchTerm) return /.*/;
```

Comment:
I actually like this the way it is. I thought I was setting it to be wild carded for all possible letters, and therefore all galleries.  Am I mistaken?